MovieChecker/MovieChecker/MovieDetailViewController.m

```diff
@@ -0,0 +1,52 @@
+//
+//  MovieDetailViewController.m
+//  MovieChecker
+//
+//  Created by Joseph Goldberg on 1/22/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import "MovieDetailViewController.h"
+
+@interface MovieDetailViewController ()
+
+@end
+
+@implementation MovieDetailViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    self.title = self.movie.name;
+    self.movieTitle.text = self.movie.name;
+    self.year.text = [NSString stringWithFormat:@"%ld", (long)self.movie.year];
```

Comment:
It's slightly preferable to use `%@` and `@`-box the value rather than cast it; i.e., `[NSString stringWithFormat:@"%@", @(self.movie.year)]`... though since you just want to turn the `NSInteger` into a string, you could call `description` on the `@`-boxed value: `[@(self.movie.year) description]`